src/rpc/blockchain.h

```diff
@@ -65,6 +66,11 @@ CBlockPolicyEstimator& EnsureAnyFeeEstimator(const std::any& context);
  * Helper to create UTXO snapshots given a chainstate and a file handle.
  * @return a UniValue map containing metadata about the snapshot.
  */
-UniValue CreateUTXOSnapshot(NodeContext& node, CChainState& chainstate, CAutoFile& afile);
+UniValue CreateUTXOSnapshot(
+    NodeContext& node,
+    CChainState& chainstate,
+    CAutoFile& afile,
+    const fs::path path,
+    const fs::path tmppath);
```

Comment:
In commit "rpc: various fixups for dumptxoutset" (b637ffaad40ea8707bdb27334990d2bcac28cf53)

Minor: Should use either `const fs::path &` or non-const `fs::path` with `std::move` to avoid creating unneeded temporary copies. `const path` just combines worst of both approaches.